### ISSUE TITLE
Scrolling down does not work well

### ISSUE THREAD

**Issue by Kohoshi12** [SimpleMobileTools/Simple-Notes#406]
I can't scroll until the final lines of a note while using the keyboard. Sometimes the scrolling on down just don't work, but sometimes it scrolls down for a few lines. It worked just fine before.

**Comment by tibbi** [comment]
what app version are you using? What settings have you changed?

**Comment by Kohoshi12** [comment]
This ones 
![Imagepipe_0](https://user-images.githubusercontent.com/74657968/107620020-67aa2f00-6c97-11eb-98aa-e63fbd9c1ace.jpg)

**Comment by tibbi** [comment]
nobody else had such issues so far. Try disabling incognito mode to test it.

**Comment by Kohoshi12** [comment]
It works fine with Incognito Mode disabled.

**Comment by tibbi** [comment]
what keyboard are you using? It might be some keyboard issue then.

**Comment by Kohoshi12** [comment]
It's OpenBoard.

**Comment by tibbi** [comment]
report it to them please, it isnt an issue on our side. It is a pretty new project so they can have such glitches.

**Comment by Kohoshi12** [comment]
No, I have the same problem with the Google Keyboard with Incognito Mode turned on.
